### PR TITLE
[FLINT]: New version

### PR DESCRIPTION
**Pull request by thofma**
This is a prerelease of 2.8.0, so I think according to our versioning scheme this should be v"200.790.000".

@fingolfin

### CODE REVIEW

File: F/FLINT/build_tarballs.jl
Comment (thofma) [inline F/FLINT/build_tarballs.jl:34]:
```suggestion
    ArchiveSource("https://github.com/wbhart/flint2/archive/refs/tags/v$(upstream_version).tar.gz", #"https://www.flintlib.org/flint-$(upstream_version).tar.gz",
```

**Comment by giordano** [comment]
Good to go or waiting for @fingolfin's seal of approval?

**Comment by thofma** [comment]
I'd rather wait before doing something bad to our ecosystem.

**Review by fingolfin — APPROVED** [review]
Looks good to me, thanks. We should be safe anyway, as this won't be used until we update at least LoadFlint

**Pull request merged**
